task1/task1.html

```diff
@@ -0,0 +1,167 @@
+                                    layout and HTML editors use Lorem Ipsum as default text, so that keyword search "lorem ipsum" immediately shows how many web pages are still waiting for his present birth. Over the years, the text Lorem Ipsum has received
+                                    a lot of versions. Some versions appeared by mistake, some - intentionally (for example, humorous versions).<sup><font size=1>1</font></sup>
+                                </td>
+                            </tr>
+                            <tr align=center>
+                                <td>
+                                    &bull; <i>Italic item<sup><a href="#"><font size=1>2</font></a></sup></i>
+                                </td>
+                            </tr>
+                            <tr align=center>
+                                <td>&bull; <b>Bold item</b></td>
+                            </tr>
+                            <tr>
+                                <td>
+                                    <b>Items in the shop</b>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>
+                                    <table border=1 cellpadding=5 cellspacing=0 width=100%>
+                                        <thead>
+                                            <th>&#8470;</th>
+                                            <th align=left>Name</th>
+                                            <th>Price</th>
+                                        </thead>
+                                        <tbody align=center>
+                                            <tr>
+                                                <td>1</td>
+                                                <td align=left>Sword</td>
+                                                <td>10</td>
+                                            </tr>
+                                            <tr>
+                                                <td>2</td>
+                                                <td align=left>Mace</td>
+                                                <td>5</td>
+                                            </tr>
+                                            <tr>
+                                                <td>3</td>
+                                                <td align=left>Faithful horse</td>
+                                                <td>15</td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </td>
+                            </tr>
+
+                        </tbody>
+                    </table>
+
+                </td>
+                <td width=200px>
+                    <table cellpadding=5>
```

Comment:
table is useless here

Reply:
Even for aligning labels and inputs in separate rows without using any CSS?